components/create-bounty-modal.tsx

```diff
@@ -38,21 +46,21 @@ import { IFilesProps } from "./drag-and-drop";
 import GithubInfo from "./github-info";
 import ReadOnlyButtonWrapper from "./read-only-button-wrapper";
 import ReposDropdown from "./repos-dropdown";
-
+import Translation from "./translation";
 
 const { publicRuntimeConfig } = getConfig();
 
 interface chainPayload {
-   title: string, 
-   cid: string | boolean,  
-   repoPath: string,
-   transactional: string,
-   branch: string
-   githubUser: string,
-   tokenAmount: number,
-   rewardToken?: string,
-   rewardAmount?: number,
-   fundingAmount?: number,
+  title: string;
```

Comment:
Can't we simply use `import {Bounty} from '@taikai/dappkit'` and use `Bounty` interface instead of having to rewrite it ?